.github/workflows/benchmark_on_pokec_large.yaml

```diff
@@ -0,0 +1,101 @@
+name: Run benchmark on pokec large dataset
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
We shouldn't run a workflow with timeout of 720 minutes on every pull request push.
On which events do we want to run this workflow, write your ideas in the reply.

Reply:
This was only for testing, I will change this part.